pkgs/by-name/do/doppler/package.nix

```diff
@@ -26,8 +27,10 @@ buildGoModule rec {
 
   nativeBuildInputs = [ installShellFiles ];
 
-  postInstall = ''
+  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```

Comment:
The conditional should probably not apply to `mv $out/bin/cli $out/bin/doppler`?

Reply:
Moved it down